src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -199,37 +195,18 @@ public override void Open()
                     break;
             }
 
-            if ((flags & SQLITE_OPEN_URI) == 0
-                && !filename.Equals(":memory:", StringComparison.OrdinalIgnoreCase)
-                && !Path.IsPathRooted(filename))
-            {
-                filename = Path.GetFullPath(Path.Combine(BaseDirectory, filename));
-            }
-
-            var rc = NativeMethods.sqlite3_open_v2(filename, out _db, flags, vfs: null);
-            MarshalEx.ThrowExceptionForRC(rc, _db);
+            var rc = raw.sqlite3_open_v2(filename, out _db, flags, vfs: null);
+            SqliteException.ThrowExceptionForRC(rc, _db);
 
             SetState(ConnectionState.Open);
         }
 
-#if !NET451
-        private static string BaseDirectory
-            => Environment.GetEnvironmentVariable("ADONET_DATA_DIR")
```

Comment:
Until we have the fix in EF tooling, can we keep this in?

I highly-suspect that this was only ever used by EF tooling, but we did document it for 1.x. It's possible this will be a breaking change for someone out there. https://github.com/aspnet/Microsoft.Data.Sqlite/wiki/Connection-Strings